src/file_storage.cpp

```diff
@@ -363,7 +363,7 @@ namespace libtorrent
 	void file_storage::add_file(std::wstring const& file, std::int64_t file_size
 		, int file_flags, std::time_t mtime, string_view symlink_path)
```

Comment:
I think it's probably *more* OK to change the signature of a deprecated function than of a non-deprecated one. This change would at most give a conversion warning in the client though, so I think it's not too bad. It's still an ABI change, but this is for a major release